inst/@sym/complexregion.m

```diff
@@ -0,0 +1,63 @@
+%% Copyright (C) 2017 Lagu
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @defmethod @@sym complexregion (@var{x}, @var{y})
+%% Represents the Set of all Complex Numbers.
+%% It can represent a region of Complex Plane in 
```

Comment:
blank line between the "lookfor" and the rest of the body.  But this looks wrong?  Should it represent all complex numbers if no arguments are passed or what does the `lookfor` summary line mean?